src/main/java/org/generationcp/middleware/api/feedback/FeedbackDto.java

```diff
@@ -0,0 +1,80 @@
+package org.generationcp.middleware.api.feedback;
+
+import org.generationcp.middleware.pojos.workbench.feedback.FeedbackFeature;
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+@AutoProperty
+public class FeedbackDto {
+
+	private Integer id;
+
+	private FeedbackFeature feature;
+
+	private String collectorId;
+
+	private Integer attempts;
+
+	private boolean enabled = true;
```

Comment:
why do you need to set a default here and why the disable and enable methods are needed? 
@cuenyad